examples/tv-app/tv-common/clusters/media-playback/MediaPlaybackManager.cpp

```diff
@@ -278,18 +278,22 @@ void MediaPlaybackManager::HandleStartOver(CommandResponseHelper<Commands::Playb
 bool MediaPlaybackManager::HandleActivateAudioTrack(const chip::CharSpan & trackId, const uint8_t & audioOutputIndex)
 {
     bool foundMatch = false;
+    std::string idString(trackId.data(), trackId.size());
     for (auto const & availableAudioTrack : mAvailableAudioTracks)
     {
-        if (strcmp(availableAudioTrack.id.data(), trackId.data()) == 0)
+        std::string nextidString(availableAudioTrack.id.data(), availableAudioTrack.id.size());
+        if (strcmp(nextidString.c_str(), idString.c_str()) == 0)
         {
             mActiveAudioTrack = availableAudioTrack;
             foundMatch        = true;
+            break;
```

Comment:
you break and then there return is basically `return foundMatch`.

how about dropping the foundMatch variable and just doing `return true` here and `return false` at the end of the function?

Reply:
done